src/server/controllers/webhook-events/github-app-install.ts

```diff
@@ -0,0 +1,72 @@
+import * as fse from 'fs-extra';
+import gql from 'graphql-tag';
+import * as path from 'path';
+
+import {RepoIdQuery} from '../../../types/gql-types';
+import {github} from '../../../utils/github';
+import {WebHookHandleResponse} from '../../apis/github-webhook';
+import {githubAppModel} from '../../models/githubAppModel';
+
+export interface InstallHook {
+  action: 'created';
+  installation: {
+    id: number; repository_selection: 'all' | 'selected';
+    permissions: {[name: string]: string;};
+    events: string[];
+    account: {login: string; avatar_url: string; type: 'User' | 'Organization';}
+    repositories: Array<{id: number; name: string; full_name: string}>
```

Comment:
Should be `,` between all these lines, and in all these objects. Only the top level `interface` uses `;`.